pytermgui/ansi_interface.py

```diff
@@ -107,243 +142,263 @@ class Color:
         "brightcyan": 14,
         "brightwhite": 15,
     }
-    """16 default named colors. Expanding this list will expand the names `pytermgui.parser.markup`
-    will recognize, but if that is your objective it is better to use
-    `pytermgui.parser.MarkupLanguage.alias`."""
+    """
+    16 default named colors.
 
-    def __init__(self, layer: int = 0) -> None:
+    Expanding this list will expand the names `pytermgui.parser.markup` will recognize, but if that is your objective it is better to use `pytermgui.parser.MarkupLanguage.alias`.
+    """
+
+    def __init__(self, layer: Literal[0, 1] = 0) -> None:
         """Initialize object
 
-        `layer` can be either 0 or 1. This value determines whether the instance
-        will represent foreground or background colors."""
+        `layer` can be either 0 or 1. This value determines whether the instance will represent foreground or background colors.
+        """
 
-        if layer not in [0, 1]:
-            raise NotImplementedError(f"Layer {layer} can only be one of [0, 1].")
+        if layer not in (0, 1):
+            raise NotImplementedError(f"Layer {layer} can only be one of (0, 1).")
```

Comment:
This makes no functional difference as far as I can tell. Using a set might have made a tiny bit more sense semantically, but the difference between lists and tuples aren't important in this case.

Reply:
Tuples are more memory efficient. It is explained in few StackOverflow Q/A